README.md

```diff
@@ -8,6 +8,29 @@
 
 A dependency manager for .NET with support for NuGet packages and git repositories.
 
+## Getting started
+
+Detailed "Getting started" guide: https://fsprojects.github.io/Paket/getting-started.html
+
+This will use the bootstrapper in [magic mode](https://fsprojects.github.io/Paket/bootstrapper.html#Magic-mode):
+
+### Windows
+
+1) Open powershell and navigate to your project directory.
+2) Execute ``[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; Invoke-WebRequest -Uri "https://get-paket.azurewebsites.net/api/bootstrapper" -OutFile ".\paket.exe"`` (the first part is required because github disabled support for older ssl versions)
```

Comment:
TODO for myself: use ``(New-Object System.Net.WebClient).DownloadFile($url, $output)`` instead, because ``Invoke-WebRequest`` relies on internet explorer ... (https://blog.jourdant.me/post/3-ways-to-download-files-with-powershell)